src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -177,6 +177,11 @@
     <PropertyGroup>
         <PackageBuildNumberOverride Condition="'$(PackageBuildNumberOverride)' == '' and '$(_DerivedPackageNumber)' != ''">$(_DerivedPackageNumber)</PackageBuildNumberOverride>
     </PropertyGroup>
+    <!-- we don't yet have runtime packages for these RIDs so exclude them for now -->
+    <ItemGroup>
+      <ExcludedRuntimes Include="fedora.23-x64" />
```

Comment:
The ones on the feed are with different version numbers because they are coming from the TFS builds. So the packages with the new build pipeline aren't self consistent when it comes to these packages yet.